src/common/components/feedback/_index.scss

```diff
@@ -16,7 +16,7 @@
 
 .feedback-container {
   position: fixed;
-  z-index: 99;
+  z-index: 999;
```

Comment:
Why it had been done? We don't use multiplying of numbers. Use +1 instead. 100 in this case